src/app/lib/config/services/tigrinya.js

```diff
@@ -55,8 +55,8 @@ const service = {
             'ነቲ ናይ ቢቢሲ መአለሺ ስፍራ ተጠቒምካ ነዚ ገጽ ክትረኽቦ ፈትን',
           ],
           callToActionFirst: 'ከም ኣማራጺ ከኣ ',
-          callToActionLinkText: '[BBC News ረኣይ።]',
-          callToActionLast: '',
+          callToActionLinkText: 'BBC News',
+          callToActionLast: ' ረኣይ።',
           callToActionLinkUrl: 'https://www.bbc.com/tigrinya',
```

Comment:
I don't think this is correct. From the translation sheet `ከም ኣማራጺ ከኣ  [BBC News [service] homepage] ረኣይ።` so then it should be

```js
callToActionFirst: 'ከም ኣማራጺ ከኣ   ',
callToActionLinkText: 'BBC News ትግርኛ homepage',
callToActionLast: ' ረኣይ።',
```

Reply:
makes sense, thx

Reply:
I agree, this is a problem accross the sheet. The service name has not been input in the brackets in many places. I assume it would be the same name as the word that appears at the top next to the BBC logo, but cannot be 100% certain there would not be a contextual language difference. Ideally, the translators woud have input the service name into the sheet. It is also different accross the sheet, where the word Homepage is sometimes ommitted, or only the word for News is used.

Reply:
From inspecting the translation sheet, I'm assuming that some languages adopt the word `homepage`

Reply:
So where the word 'homepage' is not translated, we should use the English word 'homepage' as it appears in the sheet? If we follow the sheet strictly, I think your above comment is correct and probably should be used accross all translations, including inputting the service name. However, this means we would need to go back and change some translations on the page (e.g Korean, Indonesian).

Reply:
I agree on this, but I think it is more helpful is to have exact screens as acceptance criteria for translations, so we can validate directly against. cause the sheet is bit tricky